exp/urls.py

```diff
@@ -253,10 +255,15 @@
         PreviewProxyView.as_view(),
         name="preview-proxy",
     ),
+    path(
+        "studies/preview/<int:pk>/<int:child_id>/jspsych/",
+        JsPsychPreviewView.as_view(),
+        name="preview-jspsych",
+    ),
```

Comment:
I'm not sure why, but when I try to preview an external study, I'm being redirected to the jsPsych preview URL..?

Also, (less important), I noticed that the new jsPsych study preview URL uses the study and child PK IDs, and it appends jsPsych to the end of the URL. E.g. "/exp/studies/preview/751/525/jspsych/". Whereas the EFP and external preview URLs use the study/child UUIDs and don't include anything about the study type, e.g. "/exp/studies/1fa2b85d-2438-4814-aee3-21d060b32188/60d13465-68e0-4d7a-ad57-1bb0e2329565/preview/". 

Was this an intentional decision? If not, would it be better to keep the format the same? I do like the shorter URLs, but I'm wondering if there's a benefit to using UUIDs in that it makes the valid URLs less guessable and therefore less vulnerable to spam?